kura/org.eclipse.kura.net.admin/src/main/java/org/eclipse/kura/net/admin/visitor/linux/DhcpConfigWriter.java

```diff
@@ -216,13 +189,13 @@ private void writeKuraExtendedConfig(NetInterfaceConfig<? extends NetInterfaceAd
         } else if (netInterfaceConfig instanceof WifiInterfaceConfigImpl) {
             netInterfaceAddressConfigs = ((WifiInterfaceConfigImpl) netInterfaceConfig).getNetInterfaceAddresses();
         } else {
-            s_logger.error("not adding config for " + netInterfaceConfig.getName());
+            logger.error("not adding config for " + netInterfaceConfig.getName());
```

Comment:
@markoer may be change this to  logger.error("not adding config for {}", netInterfaceConfig.getName());